deepchecks/checks/performance/class_performance_imbalance.py

```diff
@@ -0,0 +1,240 @@
+            for scorer_name, score in self._validate_results(scorer_results, len(unique_labels))
+        })
+
+        def display():
+            title = (
+                'Class Performance Imbalance Check for binary data'
+                if len(labels) == 2
+                else 'Class Performance Imbalance Check for multi-class data'
+            )
+
+            df.transpose().plot.bar(
+                title=title,
+                backend='matplotlib',
+                xlabel='Score',
+                ylabel='Values'
+            )
+
+        return CheckResult(
+            value=df.transpose().to_dict(),
+            header='Class Performance Imbalance',
+            check=type(self),
+            display=display
+        )
+
+    def _get_default_scorers(self) -> t.Dict[str, t.Callable[..., np.ndarray]]:
+        # TODO: use `get_metrics_list` from utils package
+        # but first we need to refactor it to accept 'average' argument
+        return {
+            'Accuracy': make_scorer(recall_score, zero_division=0, average=None),
+            'Precision': make_scorer(precision_score, zero_division=0, average=None),
+            'Recall': make_scorer(recall_score, zero_division=0, average=None)
+        }
+
+    def _validate_results(
+        self,
+        results: t.Iterable[t.Tuple[str, object]],
+        number_of_classes: int
+    ) -> t.Iterator[t.Tuple[str, np.ndarray]]:
+        # We need to be sure that user-provided scorers returned value with correct
+        # datatype, otherwise we need to raise an exception with an informative message
+        expected_types = t.cast(
+            str,
+            np.typecodes['AllInteger'] + np.typecodes['AllFloat'] # type: ignore
+        )
+        message = (
+            f"Check '{type(self).__name__}' expecting that scorer "
+            "'{scorer_name}' will return an instance of numpy array with "
+            f"items of type int|float and with shape ({number_of_classes},)! {{additional}}."
+        )
+
+        for scorer_name, score in results:
+            if not isinstance(score, np.ndarray):
+                raise DeepchecksValueError(message.format(
+                    scorer_name=scorer_name,
+                    additional=f"But got instance of '{type(score).__name__}'"
+                ))
+            if score.dtype.kind not in expected_types:
+                raise DeepchecksValueError(message.format(
+                    scorer_name=scorer_name,
+                    additional=f"But got array of '{str(score.dtype)}'"
+                ))
+            if len(score) != number_of_classes:
+                raise DeepchecksValueError(message.format(
+                    scorer_name=scorer_name,
+                    additional=f"But got array with shape ({len(score)},)"
+                ))
+            yield scorer_name, score
+
+    def add_condition_ratio_difference_not_greater_than(self: CP, threshold: float = 0.3) -> CP:
```

Comment:
do you think it makes sense to have single condition for many metrics? maybe we should limit the check to single metric

Reply:
do you suggest adding another parameter to the method, to narrow the condition to some specific metric?
I do not know/ @noamzbr what do you think?

Reply:
Sound reasonable to limit to a single metric

Reply:
I mean to limit the check itself, not the condition

Reply:
oh, then, did I get you right?
- instead of calculating multiple scores by default you suggest calculating only one? (which one?)
- instead of accepting dict of alternative scorers you suggest accepting only one scorer?

@noamzbr your thoughts?

Reply:
For this check we must calculate many scores. To understand imbalance I usually look at both recall and precision for e.g. In that light, I think it makes sense to limit the condition, and have the default one be on f1 (do we have f1 in the default scorers?)

Reply:
> do we have f1 in the default scorers?

nope, but I added, also updated condition method